lib/fortnox/api/repositories/base/options.rb

```diff
@@ -8,6 +8,7 @@ class Options
                         :unique_id, :attr_to_json_map, :json_to_attr_map,
                         :keys_filtered_on_save
 
+          #rubocop:disable Metrics/ParameterLists
```

Comment:
Yep. If we decide to do something about it, that would be suitable for a separate PR.